fili-core/src/main/java/com/yahoo/bard/webservice/web/responseprocessors/FullResponseProcessor.java

```diff
@@ -39,4 +47,21 @@
  *
  */
 public interface FullResponseProcessor extends ResponseProcessor {
+    @Override
+    default void processResponse(JsonNode json, DruidAggregationQuery<?> query, LoggingContext metadata) {
```

Comment:
The only way to override is to implement `processResponse` and `default` is needed to implement this method